src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonBulkCopyTest.cs

```diff
@@ -23,14 +23,17 @@ public JsonBulkCopyTest(ITestOutputHelper output)
             _output = output;
         }
 
-        private void PopulateData(int noOfRecords)
+        private void PopulateData(int noOfRecords, int rows)
         {
             using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
             {
                 DataTestUtility.CreateTable(connection, "jsonTab", "(data json)");
                 DataTestUtility.CreateTable(connection, "jsonTabCopy", "(data json)");
-                GenerateJsonFile(50000, _jsonFile);
-                StreamJsonFileToServer(connection);
+                GenerateJsonFile(noOfRecords, _jsonFile);
```

Comment:
Since you are generating this file and contents, use random file name generator.